specs/deposits.md

```diff
@@ -371,11 +371,11 @@ begins to store additional data:
 - `blobBaseFeeScalar` (`uint32`): The scalar value applied to the L1 blob base fee portion of the L1 cost.
 - `baseFeeScalar` (`uint32`): The scalar value applied to the L1 base fee portion of the L1 cost.
 
-Additionally, the `setL1BlockValues` function is deprecated and MUST never be called when the L2 block number
-is greater than the Ecotone activation block number. `setL1BlockValues` MUST be called on the Ecotone hardfork
-activation block, except if activated at genesis.
-The `setL1BlockValuesEcotone` MUST be called when the L2 block number is greater than the Ecotone hardfork
-activation block.
+Additionally, the `setL1BlockValues` function is deprecated and MUST never be called by the L1
```

Comment:
Slight nit: the L1 attributes transaction doesn't make the call necessarily, the system makes the call. It sounds like the L1 attributes transaction is an entity. Perhaps reword a bit such that it sounds like the protocol is making the call to the legacy function. Or make it sound like the tx contains calldata targeting a function

Reply:
system makes the call as directed by the data in the attributes transaction though?  I just wanted to make it clear that it wasn't calls to L1 made from the system or anyone else submitting other transactions that invoke the contract.

How about something like:

"...and the L1 attributes transaction must direct the system to invoke setL1BlockValuesEcotone instead of setL1BlockValues for blocks following the Ecotone activation block..." ?

Reply:
The transaction itself causes the EVM invocation, "directing the system to invoke" makes sense, but implies more indirection. I updated the wording to clarify the calls.